guardrails/validators.py

```diff
@@ -1874,3 +1875,286 @@ def query_vector_collection(
 
     def to_prompt(self, with_keywords: bool = True) -> str:
         return ""
+    
+@register_validator(name="provenance-v1", data_type="string")
+class ProvenanceV1(Validator):
+    """Validates that the LLM-generated text is supported by the provided contexts, by prompting another LLM.
+    'LLM-ception'!
```

Comment:
Update to `Validates that the LLM-generated text is supported by the provided contexts, by prompting another LLM (LLM-ception).`

Reply:
Complete.